File: rust/src/models/create_onboarding_link_request.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct CreateOnboardingLinkRequest {
    pub redirect_url: String,
}

impl CreateOnboardingLinkRequest {
    pub fn new(redirect_url: String) -> Self {
        Self { redirect_url }
    }
}

File: typescript/src/models/createOnboardingLinkRequest.ts
// this file is @generated

export interface CreateOnboardingLinkRequest {
  redirectUrl: string;
}

export const CreateOnboardingLinkRequestSerializer = {
  _fromJsonObject(object: any): CreateOnboardingLinkRequest {
    return {
      redirectUrl: object["redirect_url"],
    };
  },

  _toJsonObject(self: CreateOnboardingLinkRequest): any {
    return {
      redirect_url: self.redirectUrl,
    };
  },
};
